src/main/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanC100Service.java

```diff
@@ -41,9 +61,45 @@ public FormType getCaseType() {
     @Override
     public BulkScanValidationResponse validate(BulkScanValidationRequest bulkRequest) {
         // Validating the Fields..
-        return bulkScanValidationHelper.validateMandatoryAndOptionalFields(bulkRequest.getOcrdatafields(),
-                                                                          configManager.getValidationConfig(
+        Map<String, String> inputFieldsMap = getOcrDataFieldAsMap(bulkRequest.getOcrdatafields());
+
+        BulkScanValidationResponse response = bulkScanValidationHelper
+            .validateMandatoryAndOptionalFields(bulkRequest.getOcrdatafields(), configManager.getValidationConfig(
                                                                               FormType.C100));
+        List<String> manualValidationErrors = doManualValidation(inputFieldsMap);
+        if (!manualValidationErrors.isEmpty()) {
+            response.setStatus(Status.ERRORS);
+            response.getErrors().items.addAll(manualValidationErrors);
+        }
+
+        return response;
+    }
+
+    List<String> doManualValidation(Map<String, String> inputFieldsMap) {
+        List<String> errors = new ArrayList<>();
```

Comment:
Rename Method name proper actuval validation and  add java doc which have section name of form